apps/wizard/app_pages/chart_diff/chart_diff.py

```diff
@@ -241,6 +247,9 @@ def from_charts_df(
         # Get checksums
         checksums_diff = cls._get_checksums(source_session, target_session, chart_ids)
 
+        # Get all slugs from target
+        slugs_in_target = set(read_sql("SELECT slug FROM chart_slug_redirects", target_session)["slug"])
```

Comment:
Does `chart_slug_redirects` contain all chart slugs? I guess `chart_configs` does.

Reply:
Good catch! It should be actually both as it'd fail on redirected slugs, too.